### ISSUE TITLE
in PUT admin/log if level has empty value the error message is not correct

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2420]
in PUT admin/log if level has empty value the error message is not correct

#### request
```
PUT http://localhost:1026/admin/log?level=
```

#### response
```
http code: 400 - Bad Request
headers:
   date: Tue, 02 Aug 2016 13:17:07 GMT
   fiware-correlator: 69c1043c-58b3-11e6-aa2e-005056a20feb
   connection: Keep-Alive
   content-type: application/json
   content-length: 82
payload: {"error":"BadRequest","description":"Empty right-hand-side for URI param /level/"}
```

#### expected
```
http code: 400 - Bad Request
payload: {"error":"invalid log level"}
```

**Comment by fgalan** [comment]
> {"error":"invalid log level"}

That's correct. Note that /admin/log **is not** part of the NGSIv2, so it doesn't follow the NGSIv2 error response pattern. On the contrary, it uses the above, defined in the common IoTP log management API.

**Comment by kzangeli** [comment]
The problem here is that the error is caught in a neutral place.
I don't think the broker even knows that the URL path is /admin/log in the point of the source code where the error is detected.

Can easily be fixed, of course.

**Comment by fgalan** [comment]
I understand that this has been solved by PR #2465. Thus, @iariasleon could you have a look to check if this ok, please.

**Comment by iariasleon** [comment]
LGTM
```
PUT http://localhost:1026/admin/log?level=
http code: 400 - Bad Request
headers:
   Connection: Keep-Alive
   Content-Length: 55
   Content-Type: application/json
   Fiware-Correlator: c346ed44-6b7f-11e6-99ca-005056a20feb
   Date: Fri, 26 Aug 2016 11:25:16 GMT
payload: {"error":"Empty right-hand-side for URI param /level/"}
```

```
PUT http://localhost:1026/admin/log?level=sdfsdfsdf
http code: 400 - Bad Request
headers:
   Connection: Keep-Alive
   Content-Length: 29
   Content-Type: application/json
   Fiware-Correlator: e1738502-6b7f-11e6-b198-005056a20feb
   Date: Fri, 26 Aug 2016 11:26:06 GMT
payload: {"error":"invalid log level"}
```
